### PR TITLE
feat(Skeleton): new component

### PR DESCRIPTION
**Pull request by marcinsawicki**
Resolves: #889

## Description
New "Skeleton" component, divided to smaller parts:
- `SkeletonWrapper`
- `SkeletonAvatar`
- `SkeletonText`

See docs for more details 👇 

## Storybook

https://feature-889--613a8e945a5665003a05113b.chromatic.com/?path=/docs/components-skeleton--docs

## Checklist

**Obligatory:**

- [x] Self review (use this as your final check for proposed changes before requesting the review)
- [x] Add reviewers (`livechat/design-system`)
- [x] Add correct label
- [x] Assign pull request with the correct issue

### CODE REVIEW

**Review by vladko-uxds — APPROVED** [review]
👍

File: packages/react-components/src/components/Skeleton/SkeletonPlaceholder.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   * Specify the size of the element (wdith and height in px)
+   */
+  size?: number;
+  /**
+   * Specify if the animation is enabled
+   */
+  animated?: boolean;
+}
+
+export const SkeletonPlaceholder: React.FC<ISkeletonPlaceholder> = ({
```
Comment (VadymBezpalko) [inline packages/react-components/src/components/Skeleton/SkeletonPlaceholder.tsx:24]: This component is supposed to mock the avatar component? Maybe it should contain the word 'Avatar' in its name?

File: packages/react-components/src/themes/dark.scss [hunk 2]
```diff
@@ -312,4 +312,7 @@ $surface-tertiary-hover: #43434b;
   --surface-avatar-10: #00a449;
   --navbar-background: #101014;
   --tooltip-border: rgb(255 255 255 / 15%);
+  --animated-gradient-value-1: rgb(59 59 67 / 0%) 0.08%;
+  --animated-gradient-value-2: #3b3b43 50.5%;
+  --animated-gradient-value-3: rgb(59 59 67 / 0%) 99.89%;
```
Comment (VadymBezpalko) [inline packages/react-components/src/themes/dark.scss:317]: we need to add this to our TS enum, and const with groups assigning

File: packages/react-components/src/components/Skeleton/Skeleton.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+<Meta of={SkeletonStories} />
+
+<Title>Skeleton</Title>
+
+[Intro](#Intro) | [Component API](#ComponentAPI)
+
+## Intro <a id="Intro" />
+
+Skeleton is a simple component to use in the UI to present a placeholder for loading content before it is ready to be displayed in a view. It may suggest a loading avatar, a line of text, or an entire block of elements. <br /><br />
+However, please note that "Skeleton" is not a single component. This is the name for a collection of components that can be used either individually or in groups, depending on your needs.
```
Comment (VadymBezpalko) [inline packages/react-components/src/components/Skeleton/Skeleton.mdx:16]: We need to explain more explicitly when each component is supposed to be used

**Review by VadymBezpalko — APPROVED** [review]

**Pull request merged**